tests/test_models.py

```diff
@@ -1,37 +1,221 @@
 """
-Test cases for YourResourceModel Model
+Test cases for ShopCarts Model
 
 """
 import os
 import logging
 import unittest
-from service.models import YourResourceModel, DataValidationError, db
+from service import app
+from service.config import SQLALCHEMY_DATABASE_URI, SQLALCHEMY_TRACK_MODIFICATIONS
+from service.models import ShopCarts, DataValidationError, db
+from .shopCartFactory import ShopCartsFactory
```

Comment:
Could you change this to:
`from .shop_cart_factory import ShopCartsFactory`
since the name changed?